stacks/container-apps/cms-alarms.yml

```diff
@@ -0,0 +1,115 @@
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AudioSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+  ImageSqsCallbackName:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  # callback queues are being slow
+  CmsAudioCallbackSlowAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Audio Callbacks Slow"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription: Audio callbacks are behind
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 5
+      MetricName: ApproximateNumberOfMessagesVisible
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 10
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AudioSqsCallbackName
+  CmsImageCallbackSlowAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Image Callbacks Slow"
+      AlarmActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      InsufficientDataActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      OKActions:
+        - !If [CreateProductionResources, !Ref OpsErrorMessagesSnsTopicArn, !Ref OpsWarnMessagesSnsTopicArn]
+      AlarmDescription: Image callbacks are behind
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 5
+      MetricName: ApproximateNumberOfMessagesVisible
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 10
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref ImageSqsCallbackName
+  # callback queues likely aren't processing
+  CmsAudioCallbackFatalAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[CMS][Worker][SQS] ${EnvironmentType} Audio Callbacks Not Processing"
+      AlarmActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      InsufficientDataActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      OKActions:
+        - !Ref OpsFatalMessagesSnsTopicArn
+      AlarmDescription: Audio callbacks are maybe not processing
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 15
+      MetricName: ApproximateNumberOfMessagesVisible
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Average
+      Threshold: 1
```

Comment:
Not sure this is the correct setting ... if CMS ever got busy enough, we could legitimately have 1 message waiting in the queue every minute for a long time.  But how to detect "there's only 1 job, but nothing is processing at all" then?